### PR TITLE
Update api.html

### PR DESCRIPTION
**Pull request by bittlingmayer**
# Description

Fixes # _[Add the issue number]_

## Type of PR

- Creates the article _[article name]_
- Edits _[article name]_
- Other _[please describe]_

### Checklist:

- [ ] I have read the [contributing guidelines](contributing.md).
- [ ] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

**Comment by cefoo** [comment]
Should we link `adaptive` and `glossary` to their corresponding articles, or will it be too repetitive to link it in every API page?

**Comment by bittlingmayer** [comment]
I think repetitive is fine.  We should look for other concepts to link, like [languages](https://machinetranslate.org/languages).

**Comment by cefoo** [comment]
On my push:

- Fixed Liquid syntax on `api.html`.
- Added `Fine-tuning` automation on generate.py.
- Added `fine-tuning: yes/false` whenever the value is `true` on `api.yml`.

File: _layouts/api.html [hunk 1]
```diff
@@ -32,6 +35,25 @@ <h2>
...
+  <a href="/customisation">Customisation</a> is supported.
+  {% endif %}
+  {% if page.customisation contains 'Glossary' %}
+    <a href="/glossaries">Glossaries</a> can be used to customise the translation of specific words and phrases.
+  {% endif %}
+  {% if page.customisation contains 'Formality' %}
+    The API can also be customised for <a href="/formality">formality</a>.
+  {% endif %}
+  {% if page.customisation contains 'Fine-tuning' %}
+    <a href="/fine-tuning">Fine-tuning</a> on <a href="/parallel-data">parallel data</a> can be used to fine-tune the model.
```
Comment (bittlingmayer) [inline _layouts/api.html:49]: > The model can be customised by <a href="/fine-tuning">fine-tuning</a> on <a href="/parallel-data">parallel data</a>.

**Pull request merged**